### PR TITLE
enhancement: add countdown to resend verification email button

### PR DESCRIPTION
**Pull request by essofyany**

### CODE REVIEW

File: apps/frontend/src/routes/register/verify/index.tsx [hunk 1]
```diff
@@ -17,11 +18,16 @@ export const onGet: RequestHandler = async ({ cookie, redirect }) => {
   }
 };
 
+const COUNT = 59;
```
Comment (origranot) [inline apps/frontend/src/routes/register/verify/index.tsx:21]: Consider change this name to a more meaningful name

File: apps/frontend/src/routes/register/verify/index.tsx [hunk 2]
```diff
@@ -35,6 +41,22 @@ export default component$(() => {
     });
   });
 
+  useTask$(({ track }) => {
+    const resent = track(() => store.resent);
+    if (!resent) return;
```
Comment (origranot) [inline apps/frontend/src/routes/register/verify/index.tsx:46]: To make it more readable please write the return in a block.

File: apps/frontend/src/routes/register/verify/index.tsx [hunk 3]
```diff
@@ -35,6 +41,22 @@ export default component$(() => {
...
+    const interval = setInterval(() => {
+      if (store.count > 0) store.count--;
```
Comment (origranot) [inline apps/frontend/src/routes/register/verify/index.tsx:48]: same here

**Review by origranot — CHANGES_REQUESTED** [review]
Hey @essofyany great job!
It looks great, although I left some comments below.

I don't really like the design we have on the verify page, I think we should create another task to redesign it / make the verification process be more clean.
What do you think?

**Comment by essofyany** [comment]
Hi @origranot,
I agree with your feedback on the design. It indeed requires some improvements or redesigns on certain pages and components. Addressing these changes will take some time and should be handled through separate tasks.

File: apps/frontend/src/routes/register/verify/index.tsx (hunk 1 above)
Comment (essofyany) [reply apps/frontend/src/routes/register/verify/index.tsx]: Done

File: apps/frontend/src/routes/register/verify/index.tsx (hunk 2 above)
Comment (essofyany) [reply apps/frontend/src/routes/register/verify/index.tsx]: Done

File: apps/frontend/src/routes/register/verify/index.tsx (hunk 3 above)
Comment (essofyany) [reply apps/frontend/src/routes/register/verify/index.tsx]: Done

**Pull request merged**